### DISCUSSION TITLE
Unable to export changes to branch

### DISCUSSION THREAD

**Discussion by samkio** [community/community#39609] [Codespaces]
### Select Topic Area

Bug

### Body

Hi I have a Codespace I can no longer access due to surpassing storage limits. I want to export the changes to a new branch but whenever I click "Create branch" it sits there spinning forever. I've tried multiple times on different days (it worked for all other Codespaces except one).

![image](https://user-images.githubusercontent.com/575136/202931661-9b18351d-c650-4dc9-a13b-b61337553e5d.png)

**Comment by devashish2024** [comment]
Can I ask you how much space the codespace takes on [this](https://github.com/codespaces) page?

> Delete your folders you like to ignore: node_modules, .next, etc and it will reduce the size of codespace. I think exporting changes by default exports all things including cache folder like node_modules regardless of gitignore, maybe, just try?
